### ISSUE TITLE
NTR: Enterovirus RT-PCR in CSF positive (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8063]
New term request Enterovirus RT-PCR in CSF positive (samant_101213001713): Cerebrospinal fluid; reverse transcriptase polymerase chain reaction; RNA  UMLS: 1541112

**Comment by pnrobinson** [comment]
@MickeySegal
note this term encompasses both PCR and RT-PCR; it seems both are used. If this distinction is important please open a new issue.

Adding new term: 
- ID: HP:6000547
- label: Positive CSF enterovirus nucleic acid test
- parents: [Positive CSF nucleic acid pathogen test (HP:0430140)](https://hpo.jax.org/app/browse/term/HP:0430140)
- pmid: PMID:9705364
- def: Detection of enteroviral nucleic acid in the cerebrospinal fluid (CSF) by a method such as polymerase chain reaction (PCR) or reverse transcriptase (RT)-PCR.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8063
- synonyms:
